src/object-registry.js

```diff
@@ -2,19 +2,20 @@ import * as tf from '@tensorflow/tfjs-core';
 
 /**
  * ObjectRegistry stores a map of {id -> object} and a map of {id -> gc},
- * where gc denotes if we want the object of a given id to be garbage collected.
- *
- * @property {Object.<string, Plan>} objects
- * @property {Object.<string, Protocol>} gc
+ * where gc denotes if the object of given id will be garbage collected.
  */
 export default class ObjectRegistry {
+  /**
+   * @property {Object.<string, Object>} objects - a map of {id: object}
```

Comment:
Hmm, why this is moved into constructor's docblock?

Reply:
I saw Nahua's latest documentation PR and thought that it was the convention? But it should be param instead of property